internal/locale/locales/en-us.yaml

```diff
@@ -1138,6 +1138,9 @@ err_user_network_solution:
     If your issue persists consider reporting it on our forums at {{.V0}}.
 err_user_libc_solution:
   other: If you specified a glibc version, please check it is configured → [ACTIONABLE]{{.V0}}[/RESET].
+warn_revert_head:
+  other: |
+    [NOTICE]DEPRECATION WARNING[/RESET]: 'state revert HEAD' has been deprecated in favor of '[ACTIONABLE]state revert REMOTE[/RESET]'. Please use '[ACTIONABLE]state revert REMOTE[/RESET]' instead.
```

Comment:
```suggestion
    [NOTICE]DEPRECATION WARNING[/RESET]: '[ACTIONABLE]state revert HEAD[/RESET]' has been deprecated in favor of '[ACTIONABLE]state revert REMOTE[/RESET]'. Please use '[ACTIONABLE]state revert REMOTE[/RESET]' instead.
```

Reply:
FYI I intentionally did that so as to draw more attention to what we want the user to use, rather than what they just used. I will defer to you though.

Reply:
That's a good idea, I like it! My OCD was triggering at the inconsistency but there's a good reason for it so that settles it 🙏

Reply:
Okay, I've reverted the commit with your suggested changes. Please re-approve @Naatan .